holmes/core/tools.py

```diff
@@ -341,7 +341,7 @@ class Toolset(BaseModel):
     config: Optional[Any] = None
     is_default: bool = False
     llm_instructions: Optional[str] = None
-
+    type: Optional[str] = ""
```

Comment:
A. Please make it enum
B. If marked as Optional then lets make the default None.